packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -14,6 +14,7 @@ export default declare((api, options) => {
     regenerator,
     useBuiltIns,
     useESModules,
+    version: runtimeVersion = "7.0.0-beta.0",
```

Comment:
In `availableHelper`, if you omit the version, it just checks if the helper exists in the current version of core, rather than if it was historically available at a particular version. If we left this blank, we'd essentially always insert the helper imports, even when it was invalid.